src/e2e/pages/purchasePage.ts

```diff
@@ -68,6 +71,42 @@ export class PurchasePage {
     await frame.fill(".InputElement[name=postal]", "77777");
   }
 
+  async fillOutPaypalInfo(context: BrowserContext) {
+    const emailToUse = process.env.E2E_TEST_PAYPAL_LOGIN as string;
+    const pwdToUse = process.env.E2E_TEST_PAYPAL_PASSWORD as string;
+    expect(emailToUse).not.toBeUndefined();
+    expect(pwdToUse).not.toBeUndefined();
+
+    //away from mozilla
+    const pagePromise = context.waitForEvent("page");
+    await this.paypalButton.click();
+    const newPage = await pagePromise;
+    await newPage.waitForLoadState();
+    await newPage.waitForURL(/.*paypal\.com\/checkoutnow.*/);
+
+    const emailPrompt = newPage.locator("#email");
```

Comment:
just wanted to confirm, are these id's the same when you go through the flow in prod as well?